pyfar/classes/transmission_matrix.py

```diff
@@ -0,0 +1,395 @@
+            identical to the entries of the T-matrix, i.e. shape(tmat.A) == shape(TF).
+
+        """
+        return 1 / (self.A + self.B / Zl)
+
+    def transfer_function_quantity2(self, Zl: complex | FrequencyData):
+        """Returns the transfer function of the second quantity (output/input).
+
+        The TF is calculated based on the load impedance at the output. The second
+        quantity usually refers to some flow or movement, e.g. current, velocity,...
+        See Equation (2-1) in Reference [1]_: Defined as i2/i1 using e2 = i2*Zl.
+
+        Parameters
+        ----------
+        Zl : complex | FrequencyData
+            The load impedance data. The shape must match the entries of the T-matrix,
+            i.e. shape(tmat.A) == shape(Zl) or must be broadcastable.
+
+        Returns
+        -------
+        TF : FrequencyData
+            A FrequencyData object with the resulting transfer function. The shape is
+            identical to the entries of the T-matrix, i.e. shape(tmat.A) == shape(TF).
+
+        """
+        return 1 / (self.C * Zl + self.D)
+
+    # def transfer_function_quantity1_to_quantity2(self, Zl):
+    #     pass
+    # def transfer_function_quantity2_to_quantity1(self, Zl):
+    #     pass
+
+    @classmethod
+    def _create_abcd_matrix_broadcast(cls, abcd_data: FrequencyData,
+                                      abcd_cshape : tuple):
+        if abcd_data.cdim != 2 or abcd_data.cshape != (2,2):
+            raise ValueError("'abcd_data' must have a cshape of (2,2).")
+
+        if abcd_cshape != ():
+            abcd_data.freq = np.broadcast_to(abcd_data.freq,
+                                np.append(abcd_cshape, abcd_data.freq.shape))
+        return cls(abcd_data.freq, abcd_data.frequencies)
+
+    @classmethod
+    def create_identity(cls, frequencies, abcd_cshape = ()):
+        """Creates an object with identity matrix entries (bypass).
+
+        See Equation (2-7) in Table I of Reference [1]_.
+
+        Parameters
+        ----------
+        frequencies : array_like
+            The frequency sampling points in Hz.
+        abcd_cshape : tuple | int, optional
+            Shape of additional dimensions besides (2,2,num_bins).
+            Per default, no additional dimensions are created.
+
+        Returns
+        -------
+        cls : TransmissionMatrix
+            A TransmissionMatrix object that contains 2x2 identity matrices.
+
+        """
+        identity = np.array([[1,0],[0,1]])
+        identity_3d = np.repeat(identity[:, :, np.newaxis], len(frequencies), axis = 2)
+        return cls._create_abcd_matrix_broadcast(
+            FrequencyData(identity_3d, frequencies), abcd_cshape)
+
+    @classmethod
+    def create_series_impedance(cls, impedance : FrequencyData, abcd_cshape = ()):
+        """Creates a transmission matrix representing a series impedance.
+
+        This means the impedance is connected in series with a potential load impedance.
```

Comment:
Maybe add math equation for illustration?

Reply:
Done